mobile_files/package.json.orig

```diff
@@ -15,7 +15,7 @@
     "assert": "1.4.1",
     "asyncstorage-down": "4.0.1",
     "babel-preset-react-native": "5.0.2",
-    "bignumber.js": "https://github.com/status-im/bignumber.js.git",
+    "bignumber.js": "github:status-im/bignumber.js#master",
```

Comment:
I would prefer making a tag/release in our fork and stick to it instead of having `master` branch there.

Reply:
Yeah, also, please don't drop the `.git` from it. I've seen before both `npm` and `yarn` issues caused by this.

Reply:
> Yeah, also, please don't drop the `.git` from it. I've seen before both `npm` and `yarn` issues caused by this.

Ah, I just copied what was on desktop to make them the same, didn't notice that. I'll add it back.

Reply:
Moving this specific commit to a future PR since it involves changes in dependencies.

Reply:
> I would prefer making a tag/release in our fork and stick to it instead of having master branch there.

@mandrigin I have the impression that most of our customized npm dependencies don't follow that (they either point to a branch or to a sha). Is it really a problem for our purposes. AFAICS if someone calls `yarn install` it will retrieve the last resolved sha even if the branch moved. It is only when you call `yarn upgrade` that it will be problematic because it will now be pointing to a newer commit. Am I missing something?

Reply:
it won't help from history rewrites if we still want to build older versions. otherwise, it is okay, yes.